pkg/github/actions.go

```diff
@@ -0,0 +1,99 @@
+package github
```

Comment:
There are a *lot* of Actions APIs, I suspect the single-file per API area will break down at some point.

I tried defining a separate `actions` package, but it created some circular dependencies with this `github` package.
I'd be happy to explore that more, but I wanted to keep these changes focused on this tool.

Reply:
Appreciate you mentioning though, it's same reason I didn't consider new packages for tools yet.

Pretty sure the helpers and server can be extracted to own packages to avoid this.

CC @williammartin @juruen